docs/releases/v3.0.1.rst

```diff
@@ -0,0 +1,68 @@
+..
+    This file is part of Invenio.
+    Copyright (C) 2018 CERN.
+
+    Invenio is free software; you can redistribute it and/or modify it
+    under the terms of the MIT License; see LICENSE file for more details.
+
+Version 3.0.1
+=============
+
+Invenio v3.0.1 fixes several bugs in Invenio v3.0.0.
+
+Bug fixes
+---------
+
+- **Invenio-Access:** Fixed bug with the ``any_user``-need not being provided
+  by the anonymous identity.
+
+- **Invenio-App:** Fixed compatibility issue with latest release of
+  Flask-Talisman.
+
+- **Invenio-DB:** Fixed a compatibility issue with newly released
+  SQLAlchemy-Continiuum.
+
+- **Invenio-Search:** Fixed a compatibility issue with
+  elasticsearch-dsl>=6.2.0.
+
+- **Pytest-Invenio:** Fixed a problem which caused Invenio modules to be loaded
```

Comment:
Maybe grouping these might look a bit better and be easier to read? (e.g. like Django does on a per-"module" level: https://docs.djangoproject.com/en/2.1/releases/2.1/#minor-features)

Reply:
Actually they keep it flat for patch releases: https://docs.djangoproject.com/en/2.1/releases/2.1.1/

My thinking was just that in most cases there's one or two bullets per module.